### PR TITLE
feature: syncleaf CRD and informer init

### PR DESCRIPTION
**Pull request by rxy0210**
#### What type of PR is this?
/kind feature

#### What does this PR do?

feature: syncleaf CRD and informer init
#### Which issue(s) does this PR fix?

Fixes #
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/apis/kosmos/v1alpha1/syncleaf_types.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:object:generate=true
+// +kubebuilder:storageversion
+// +kubebuilder:rbac:groups=velero.io,resources=backups,verbs=create;delete;get;list;patch;update;watch
+// +kubebuilder:rbac:groups=velero.io,resources=backups/status,verbs=get;update;patch
+
+// SyncLeaf is custom resource that represents the capture of sync leaf cluster
+type SyncLeaf struct {
+	metav1.TypeMeta `json:",inline"`
```
Comment (duanmengkk) [inline pkg/apis/kosmos/v1alpha1/syncleaf_types.go]: SyncLeafPolicy or PromotePolicy ？

**Comment by duanmengkk** [comment]
/cc @yuleichun-striving 
/cc @village-way 
We need to research how to use Prow's label command to trigger the re-execution of GitHub Action.

**Comment by village-way** [comment]
> /cc @yuleichun-striving /cc @village-way We need to research how to use Prow's label command to trigger the re-execution of GitHub Action.

```
on:
  push:
    tags:
      - '*'
  pull_request:

```
in github action , the workflow config above, means push and pull_request will trigger the action, so we can use git push tag to the repo to trigger the action

File: cmd/clustertree/cluster-manager/app/manager.go [hunk 2]
```diff
@@ -234,6 +234,13 @@ func run(ctx context.Context, opts *options.Options) error {
 		return fmt.Errorf("error starting root pvc controller %v", err)
 	}
 
+	//promotePolicyReconciler := controllers.PromotePolicyReconciler{
```
Comment (duanmengkk) [inline cmd/clustertree/cluster-manager/app/manager.go]:
```suggestion
	//promotePolicyController := controllers.promotePolicyController{
```

Comment (rxy0210) [reply cmd/clustertree/cluster-manager/app/manager.go]: fixed

**Pull request merged**